### PR TITLE
#159987713 Implement Email notification functionality

### PR DESCRIPTION
**Pull request by lauragift21**
#### What does this PR do?
It enables the ability for users to receive email notification for interactions with the app.
#### Description of Task to be completed?
- Users should receive email notification for when there's a newly published article
- Users should receive email notification when there's interaction on a comment
- Write tests for email notification

#### How should this be manually tested?
- When the `api/articles` endpoint is triggered to create an article all followers of the user gets an email notification.
- When the `api/articles/:slug/comments/:comment.id` endpoint is triggered to send a reply to a comment all the participant of the comment gets an email notification.

#### What are the relevant pivotal tracker stories?
[159987713](https://www.pivotaltracker.com/story/show/159987713)

#### Screenshots (if appropriate)
img
img

### CODE REVIEW

**Review by CEOehis — APPROVED** [review]
LGTM.

File: server/tests/utils/article.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+import { describe } from 'mocha';
```
Comment (emmaadesile) [inline server/tests/utils/article.test.js]: I don't think you have to `import { describe } from 'mocha'`

File: server/tests/utils/comment.test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+import { describe } from 'mocha';
```
Comment (emmaadesile) [inline server/tests/utils/comment.test.js]: No need to `import { describe }  from 'mocha'`
You can use it without importing it

**Review by emmaadesile — CHANGES_REQUESTED** [review]

File: server/tests/utils/article.test.js (hunk 1 above)
Comment (lauragift21) [reply server/tests/utils/article.test.js]: Alright thanks, I'll make an edit to the file.

**Review by emmaadesile — APPROVED** [review]

**Pull request merged**